libpod/container.go

```diff
@@ -254,6 +254,8 @@ type ContainerNamedVolume struct {
 	// IsAnonymous sets the named volume as anonymous even if it has a name
 	// This is used for emptyDir volumes from a kube yaml
 	IsAnonymous bool `json:"setAnonymous,omitempty"`
+	// SubPath determines which part of the Source will be mounted in the container
+	SubPath string
```

Comment:
I think it does since in generateSpec (which play kube eventually calls) we use `ContainerNamedVolume` to generate the mountpoint and we need to subpath to get the source properly.

Reply:
Shouldn't we just put the full path (path + subpath) into the named volume as Path?

Reply:
Oh, wait, I see the problem there, different containers will want to index into the volume at different places.